assets/styles/_base.scss

```diff
@@ -481,6 +464,32 @@ select {
   }
 }
 
+.select {
+  display: inline-block;
+  position: relative;
+  vertical-align: top;
+
+  &::after {
+    border-bottom: 2px solid var(--theme-title-color);
+    border-left: 2px solid var(--theme-title-color);
+    content: '';
+    height: 8px;
+    position: absolute;
+    right: 20px;
+    top: calc(50% - 6px);
+    transform: rotate(-45deg);
+    width: 8px;
+    z-index: 1;
+  }
+
+  select {
+    // don't change backgound, if you need to change it use wrapper .select
+    background: none;
+    position: relative;
+    z-index: 2;
+  }
+}
```

Comment:
I assume that this is enabling the small icon to take the color from the customizer. Looks like a great solution! Did you check if it works with the Contact Form 7 form? Does the form component need changes?

Reply:
yes, it work's with Contact Form 7. The component itself doesn't need any specific changes (just message with forms examples need small adjustment, that I already did and pushed to premium components repo). And In the backend in Contact Forms template we need to wrap select into span with .select class. That's it
![image](https://user-images.githubusercontent.com/7567926/87560203-802f7800-c6bb-11ea-8b4f-386a71ca0ed6.png)

Reply:
I just had a fresh look on this and have 2 considerations about the change:
1. This changes the default markup for the select element, which needs to be learned by people using Flynt. We should prevent people having to learn new things whenever there is no big advantage to it.
2. Default browser UI elements (like date picker icon) have a dark color and don't change based on the background color. Changing the whole form UI to a dark background, would need other elements to be changed as well to look perfect. However, I expect them to still be usable even with the dark elements.

That's why I propose to stay with the default browser markup and a dark icon for all themes.

Reply:
select has fallback with dark icon if person forget to wrap select in .select